tests/Functional/BeaumontConsultingTest.php

```diff
@@ -3,55 +3,33 @@
 namespace Tests\Functional;
 
 use EthicalJobs\Sanitize\PurifierFactory;
+use Tests\Fixtures\LoadsFixtures;
 use Tests\TestCase;
 
 class BeaumontConsultingTest extends TestCase
 {
-    /**
-     * @test
-     */
-    public function it_correctly_purifies_fixture_1()
-    {
-        $before = file_get_contents(__DIR__ . '/../Fixtures/BeaumontConsulting/fixture.1.before.html');
-
-        $after = file_get_contents(__DIR__ . '/../Fixtures/BeaumontConsulting/fixture.1.after.html');
-
-        $purifier = PurifierFactory::create();
-
-        $purified = $purifier->purify($before);
-
-        $this->assertEquals($purified, $after);
-    }
+    use LoadsFixtures;
 
     /**
-     * @test
+     * Data for it_correctly_purifies test
      */
-    public function it_correctly_purifies_fixture_2()
+    public final function data(): iterable
```

Comment:
Do all these test methods need to be `final`? Is there a risk of the method being overridden?

Reply:
It's my strict setting in my IDE, I can't find where it is :). Will turn this off and remove all of them as it is a bit full on.